packages/components/src/checkbox-control/style.scss

```diff
@@ -27,16 +27,17 @@ $checkbox-input-size-sm: 24px; // Width & height for small viewports.
 	@include reduce-motion("transition");
 
 	&:focus {
-		box-shadow: 0 0 0 ($border-width * 2) $white, 0 0 0 ($border-width * 2 + $border-width-focus) var(--wp-admin-theme-color);
+		box-shadow: 0 0 0 ($border-width * 2) $white, 0 0 0 ($border-width * 2 + $border-width-focus) $components-color-accent;
+		border-color: $components-color-accent;
```

Comment:
`border-color` was added because without it, the focused input box retained the default `--wp-admin-theme-color` border color:

<table>
<tr>
	<td>Before:</td>
	<td>After:</td>
</tr>
<tr>
	<td>
		<img width="172" alt="Screen Shot 2022-10-26 at 07 43 30" src="https://user-images.githubusercontent.com/13856531/198018137-45a80c59-355d-4ccc-bfdb-4d313d65e28b.png">
	</td>
	<td>
		<img width="166" alt="Screen Shot 2022-10-26 at 07 43 09" src="https://user-images.githubusercontent.com/13856531/198018177-25722e11-5de0-471a-927c-109d2c854195.png">
	</td>
</tr>
</table>

Reply:
On second thought, as I continue working this doesn't feel like the right solution. I'm seeing other, similar border issues (e.g. on `FormTokenField`).

They look suspiciously like the `focus` styles in `utils/input/base.js` (which will be updated during the course of this PR), but updating the vars in that file doesn't seem to have an impact. Once I sort out what's actually going on, this added `border-color` declaration will probably be removed.

Reply:
After some guidance from @ciampo, was able to identify the `@wordpress/base-styles mixins` that were at the root of these colors. I've also added `&[aria-checked="mixed"]` to the CheckboxControl's styles, as that was also using the `--wp-admin-theme-color` in the mixin.

Reply:
@mirka this may be a recurring theme (pun not intended) — some styles may come from the `base-styles` package, and we'll have to decide if we should override them in the components package with our own theme variable, or if we should just leave them as they are.

Reply:
There were a few more of these as well. I've added comments and/or isolated those changes in separate commits so they're easy to remove if we decide to leave them alone for the time being.

Reply:
Interesting! I played around with how CSS variable reassignment works to see if it would suffice for our use case, but unfortunately it all becomes circular so it won't work 😂

```scss
.component-foo {
  // It would've been nice if the var() was evaluated before assignment, but it doesn't work like that 🙃
  --wp-admin-theme-color: var(--wp-admin-theme-color, somefallback);

  @include some-base-styles-mixin;
}
```

I am actually now considering decoupling wp-components from some of the base-styles mixins (i.e. copy/paste the relevant stuff over). The mixin dependencies were already hard to work with in wp-components, not to mention being a blocker for Emotion migration. In this CheckboxControl for example, you can see some of the styles are duplicated for some/no reason. And the fact that WP's "canonical" CheckboxControl styles have already diverged from the base-styles mixin means that the mixin isn't useful for style consistency anyway 🤷

tl;dr — For this PR I am fine with leaving the base-styles overrides out, so we can address them in a coordinated way later.

Reply:
> tl;dr — For this PR I am fine with leaving the base-styles overrides out, so we can address them in a coordinated way later.

Should we have a tracking issue where we list the overrides? Or do you think that the problem is so widespread that it's a wasted effort?

Reply:
> Should we have a tracking issue where we list the overrides? Or do you think that the problem is so widespread that it's a wasted effort?

If it's useful for task coordination, sure. I think it's mostly just `@include input-control`.

Reply:
Cool. I also proposed it since Chad mentioned that there are a few instances where he had to add such overrides:

> I've added comments and/or isolated those changes in separate commits so they're easy to remove if we decide to leave them alone for the time being.